### ISSUE TITLE
Microsoft Dynamics 365: Could not find a property named 'fullname' on type 'Microsoft.Dynamics.CRM.campaign'

### ISSUE THREAD

**Issue by jamesmacwhite** [verbb/formie#1185]
### Describe the bug

When using the refresh integration option, the following is reported by the plugin:

>API error: “{"error":{"code":"0x0","message":"Could not find a property named 'fullname' on type 'Microsoft.Dynamics.CRM.campaign'."}}” /app/vendor/guzzlehttp/guzzle/src/Exception/RequestException.php:113

### Steps to reproduce

1. Choose a form
2. Trigger the refresh integration action from the control panel
3. After a while a 500 error occurs with the above details.

### Form settings

N/A

### Craft CMS version

3.7.61

### Plugin version

1.6.19

**Comment by jamesmacwhite** [comment]
The docs suggest fullname doesn't exist in the context:

https://learn.microsoft.com/en-us/dynamics365/customerengagement/on-premises/developer/entities/campaign?view=op-9-1

Possibly should be `name`

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.6.19",
  "...": "..."
}
```

Then run `composer update`.

**Comment by jamesmacwhite** [comment]
Thanks!

**Comment by jamesmacwhite** [comment]
Just tested with dev-craft-3 and refresh integration works now!

**Comment by engram-design** [comment]
Fixed in [1.6.20](https://github.com/verbb/formie/releases/tag/1.6.20)
